mungegithub/mungers/submit-queue.go

```diff
@@ -1331,6 +1335,10 @@ func (sq *SubmitQueue) handleGithubE2EAndMerge() {
 }
 
 func (sq *SubmitQueue) mergePullRequest(obj *github.MungeObject, msg, extra string) bool {
+	isMaster, _ := obj.IsForBranch(branchMaster)
+	if isMaster {
+		extra = extra + ". " + sq.MergeToMasterMessage
```

Comment:
Since this function is called from a goroutine that is not the main goroutine, the `opts` lock must be held to read `sq.MergeToMasterMessage` like [here](https://github.com/kubernetes/test-infra/blob/master/mungegithub/mungers/submit-queue.go#L872) in order for this to be thread safe.